Sprint-2/errors/1.js

```diff
@@ -3,11 +3,14 @@
 // Why will an error occur when this program runs?
 // Try playing computer with the example to work out what is going on
```

Comment:
What do you think? Why is this happening? What is the fix?

Reply:
Remove the duplicated declaration. With [this commit ](https://github.com/CodeYourFuture/Module-Structuring-and-Testing-Data/pull/146/commits/17d1618436ef72fbdd53e66334675b2dfb7fe2ce)